cobalt/audio/audio_helpers.h

```diff
@@ -68,7 +68,10 @@ inline size_t GetSampleTypeSize(SampleType sample_type) {
 // an internal SampleType.  If we are not running on starboard or using the
 // starboard media pipeline, then the preferred sample type is always float32.
 inline SampleType GetPreferredOutputSampleType() {
-#if defined(STARBOARD)
+#if SB_API_VERSION > 15
```

Comment:
Everything LGTM, but there's a style question. I find it easier to reason about "changes in Starboard16" if the conditionals also compare it to SB16. E.g.

```
#if SB_API_VERSION >= 16
```
rather than
```
#if SB_API_VERSION > 15
```

I don't think it significantly matters either way though, and i think i've seen both styles used elsewhere.

Reply:
Done